### PR TITLE
Misc improvements

### PR DESCRIPTION
**Pull request by MengZhuNAV**
### 💰 Hva forsøker du å løse i denne PR'en
A couple of improvements/bugfix:
 + Do not show Id of avsender when it is changed
 + Change style of some buttons
 + Use uppercase for the first letter that is followed by lowercases for user/avsender names
 + Refine document title list
 + Add a separating line between title groups

### ✅ Checklist
_Har du husket alle punktene i listen?_
- [ ] Jeg har testet mine endringer i henhold til akseptansekriteriene 🕵️
- [ ] Jeg har skrevet tester. Hvis du ikke har skrevet tester, beskriv hvorfor under 👇

_Jeg har ikke skrevet tester fordi:_

### 🤷‍♀ ️Hvor er det lurt å starte?
alt i ett

### 💬 Ønsker du en muntlig gjennomgang?
- [ ] Ja
- [X] Nei
  
### 👀 Screen shots
_Har det visuelle endret seg? Legg til før- og etterbilder!_

### CODE REVIEW

File: src/frontend/context/ManuellJournalførContext.tsx [hunk 1]
```diff
@@ -375,13 +375,14 @@ const [ManuellJournalførProvider, useManuellJournalfør] = createUseContext(()
...
 
-    const settAvsender = (navn: string) => {
+    const settAvsender = (navn: string, id = '') => {
         const oppdatert = { ...oppdatertData };
         if (
             oppdatert.status === RessursStatus.SUKSESS &&
             oppdatert.data.journalpost.avsenderMottaker
         ) {
             oppdatert.data.journalpost.avsenderMottaker.navn = navn;
+            oppdatert.data.journalpost.avsenderMottaker.id = id;
```
Comment (henninghaakonsen) [inline src/frontend/context/ManuellJournalførContext.tsx:385]: What is id in this context?

File: src/frontend/utils/formatter.ts [hunk 2]
```diff
@@ -73,3 +73,17 @@ export const sisteDatoIMnd = (måned: number, år: number): Date => {
 
 export const sorterFødselsdato = (fødselsDatoA: string, fødselsDatoB: string) =>
     familieDayjs(fødselsDatoA).isBefore(fødselsDatoB) ? 1 : -1;
+
+export const formaterTilKunFørstBokstavStor = (navn: string | undefined): string | undefined => {
```
Comment (henninghaakonsen) [inline src/frontend/utils/formatter.ts:77]: Is this for the name? We don't change the capitalization in the app, and should not do this here either.

File: src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx [hunk 3]
```diff
@@ -137,7 +139,8 @@ export const KnyttJournalpostTilBehandling: React.FC = () => {
                                         }}
                                     />
                                 </StyledTd>
-                                <td>{behandling.status}</td>
+                                <td>{formaterTilKunFørstBokstavStor(behandling.status)}</td>
```
Comment (henninghaakonsen) [inline src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx:142]: You can use .capitalize() for this I think 👍

**Review by henninghaakonsen — APPROVED** [review]
Nice, just a couple of comments :)

File: src/frontend/context/ManuellJournalførContext.tsx (hunk 1 above)
Comment (MengZhuNAV) [reply src/frontend/context/ManuellJournalførContext.tsx:385]: It should be Fnr if the avsender is a person, and I do not know what it means for institution

File: src/frontend/utils/formatter.ts (hunk 2 above)
Comment (MengZhuNAV) [reply src/frontend/utils/formatter.ts:77]: I agree, but Gunn is asking for this change. I think she has considerations from designer point of view

File: src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx (hunk 3 above)
Comment (MengZhuNAV) [reply src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx:142]: What can .capitalize() do? I searched but did not find the function. I need to capitalize the first letter of ALL words in a name, and is the function capable of doing it?

**Pull request merged**